requirements.txt

```diff
@@ -1,2 +1,3 @@
 pbr>=1.6
 requests
+jinja2
```

Comment:
This requirements file is used for all the ci-framework, what means that jinja2 (that is already in use, but embedded in ansible) will be added there too, that's why I preferred to try to not to use external libs. Anyway, we can get read of this by launching the script in a properly configure container using podman, like other scripts in the framework.

Reply:
+1 - I'd rather use `f'foo {blah}'` or any core templating feature.